tests/parser/class-abstract.ts.fixture.ts

```diff
@@ -1,5 +1,4 @@
 class Foo {
-  bar(): void;
+  abstract bar(): void;
 }
-// ERROR 1042: "'abstract' modifier cannot be used here." in class-abstract.ts(2,3+8)
-// ERROR 2391: "Function implementation is missing or not immediately following the declaration." in class-abstract.ts(2,19+4)
+// ERROR 1244: "Abstract methods can only appear within an abstract class." in class-abstract.ts(2,3+8)
```

Comment:
should I include the `Function implementation is missing` error here or is this good?